### ISSUE TITLE
DUP: Duplicate of #125 - RTL text not showing in the result blocks, but works well in the commandline.

### ISSUE THREAD

**Issue by michbarsinai** [warpdotdev/warp#279]
### Describe the bug

```shell
When a result block contains Hebrew/Arabic text, the test is not shown. Not sure if it's the right-to-left or the face that it's non-ascii.
```

### To Reproduce

Steps to reproduce:
1. Go to a regular warp terminal window
2. type `echo hello שלום أهلا`
3. type `enter`

### Expected behaviour

```shell
Expected behavior is to see `hello שלום أهلا` on the result block. In reality, only `hello` is shown.
```

### Screenshots

<img width="549" alt="image" src="https://user-images.githubusercontent.com/1938468/137600748-f8f0df0e-ea4b-4dc6-bb81-59d182e8e503.png">

Warp on the left, Apple Terminal on the right.

### Operating System

MacOS

### OS Version

11.6

### Additional context

```shell
Might be interesting
```

**Comment by zheng** [comment]
one of our eng's is working on this & has a working prototype! I think it will be out in another release or two. stay tuned!

**Comment by elviskahoro** [comment]
Linking:
- #125

**Comment by BSN4** [comment]
this is being fixed in PR #10809, before and after screenshots on the PR, tracking issue is #10355
